config-example/config.txt

```diff
@@ -1,13 +1,13 @@
 ## Server name: This appears at the top of the screen in-game. In this case it will read "tgstation: station_name" where station_name is the randomly generated name of the station for the round. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice
-# SERVERNAME tgstation
+# SERVERNAME Derelict 12
 
 ## Alert levels
-ALERT_GREEN All threats to the station have passed. Security may not have weapons visible, privacy laws are once again fully enforced.
-ALERT_BLUE_UPTO The station has received reliable information about possible hostile activity on the station. Security staff may have weapons visible, random searches are permitted.
-ALERT_BLUE_DOWNTO The immediate threat has passed. Security may no longer have weapons drawn at all times, but may continue to have them visible. Random searches are still allowed.
-ALERT_RED_UPTO There is an immediate serious threat to the station. Security may have weapons unholstered at all times. Random searches are allowed and advised.
-ALERT_RED_DOWNTO The self-destruct mechanism has been deactivated, there is still however an immediate serious threat to the station. Security may have weapons unholstered at all times, random searches are allowed and advised.
-ALERT_DELTA The station's self-destruct mechanism has been engaged. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.
+ALERT_DAYBREAK Daybreak protocol reassigned. No immediate threats detected, crew may continue to carry out regular work. Sunset Division are to return to standard posts and await further instruction.
+ALERT_SUNSET_UPTO Sunset protocol initiated. Possible or minimal threat detected, Any and all suspicious activity should be reported to security. Crew may carry out regular work with maximum sensors activated, Extra vehicular activity is prohibited.
+ALERT_SUNSET_DOWNTO Sunset has returned. Nightfall protocols are rescinded. Please continue your shift with caution, and comply with Asset Protection's requests.
+ALERT_NIGHTFALL_UPTO Nightfall protocol initiated, immediate threat confirmed. Crew rights have been limited, all personnel are to be secluded to their respective departments. Failure to comply will result in jail sentence and immediate relief of duty! This is not a drill.
+ALERT_NIGHTFALL_DOWNTO Supernova procedure has been averted; standard law protocol has been reinstated. All crew are to return to their departments or proceed with evacuation in an orderly manner..
```

Comment:
"Supernova procedure has been averted; standard Nightfall protocol has been reinstated. All crew are to return to their departments or proceed with evacuation in an orderly manner."